### ISSUE TITLE
4x maps

### ISSUE THREAD

**Issue by Timmiej93** [pseudex/Coursedrawer#6]
I believe, based on my short testing, that 4x maps don't work yet. When loading the PDA map and loading a course, it doesn't match, the map is obviously scaled down.
As a suggestion: Maybe just have a button to manually tell the program this is a 4x map, so all coordinates (or the background map) should be scaled?

**Comment by LordKena** [comment]
The map is 2048x2048 and you need to resize it too 4096x4096

**Comment by Timmiej93** [comment]
Yes, but that doesn't fix the issue that the points of the course actually are outside of the program, as you can see in the picture I attached.

![image](https://user-images.githubusercontent.com/16491748/27013040-30f26068-4edc-11e7-823e-1d20a02776f3.png)

**Comment by StephanSteinert** [comment]
Anything new with this issue?
